antidox/.pylintrc

```diff
@@ -0,0 +1,4 @@
+[FORMAT]
```

Comment:
inside google we can just use
pyformat -i foo.py
and
gpylint
so no need to check in this file.

Reply:
Discussed with Jeff offline, we think we should keep this change for now since this is an open source project we can encourage people to use the public pylint tool just with slightly modified settings